namerd/storage/zk/src/main/scala/com/twitter/finagle/serverset2/buoyant/ZkDtabStore.scala

```diff
@@ -0,0 +1,127 @@
+package com.twitter.finagle.serverset2.buoyant
```

Comment:
It would be nice to add a comment indicating why this needs to be in the `com.twitter.finagle` package (I assume because we make use of some package-private code?)